demo/data/data.json

```diff
@@ -0,0 +1,2500 @@
+[
```

Comment:
Since we're going to want this data across contexts and know we're using npm, maybe we should make a `jupyter-test-data` package on `npm`.